include/glow/Base/Tensor.h

```diff
@@ -47,6 +47,15 @@ ShapeVector expandDimsToMax(llvm::ArrayRef<size_t> currDims);
 
 /// A class that represents a contiguous n-dimensional array (a tensor).
 class Tensor final {
+public:
+  /// Specifies the kind initialization for the tensor.
+  enum class InitKind {
+    Zero,      // The tensor is initialized to zero.
```

Comment:
It seems like Zero is a special case of Broadcast with 0 value. Do we really need to have this extra init type for tensor?

Reply:
You are right. This part will go away when we finish the refactoring. I am trying to minimize code changes in this massive refactoring.